app/src/main/java/org/coepi/android/ui/extensions/PublicReportExtensions.kt

```diff
@@ -0,0 +1,56 @@
+package org.coepi.android.ui.extensions
+
+import org.coepi.android.R.string
+import org.coepi.android.api.publicreport.CoughSeverity
+import org.coepi.android.api.publicreport.CoughSeverity.DRY
+import org.coepi.android.api.publicreport.CoughSeverity.EXISTING
+import org.coepi.android.api.publicreport.CoughSeverity.NONE
+import org.coepi.android.api.publicreport.CoughSeverity.WET
+import org.coepi.android.api.publicreport.FeverSeverity
+import org.coepi.android.api.publicreport.FeverSeverity.MILD
+import org.coepi.android.api.publicreport.FeverSeverity.SERIOUS
+import org.coepi.android.api.publicreport.PublicReport
+import org.coepi.android.system.Resources
+
+fun PublicReport.AlertDetailToUIString(resources: Resources) : String {
```

Comment:
What happened with the bullets? If this is specific to details, move to the view model (and make the other functions non-private).